lib/smart_answer_flows/calculate-agricultural-holiday-entitlement.rb

```diff
@@ -35,11 +35,9 @@ def define
         option "2-days"
         option "1-day"
 
-        calculate :days_worked_per_week do |response|
-          # XXX: this is a bit nasty and takes advantage of the fact that
-          # to_i only looks for the very first integer
-          response.to_i
-        end
+        # XXX: this is a bit nasty and takes advantage of the fact that
+        # to_i only looks for the very first integer in the string response
+        calculate :days_worked_per_week, &:to_i
```

Comment:
I agree this looks a bit odd. I added "in the string response" to try to make things clearer, but it sounds as if that wasn't enough. To address your concern I'm going to leave the code unchanged and disable the cop for these lines. Hopefully that will highlight how undesirable this code is and it will encourage us to remove the calculate block as you suggest.

Addressed in 4c23b45e9db97580079e5980b9e6d9f2200ce41f.